### ISSUE TITLE
Windows 11 - WSL2 takes 10+ minutes to launch

### ISSUE THREAD

**Issue by plasma-vision** [microsoft/WSL#11692]
### Windows Version

Microsoft Windows [Version 10.0.22631.3737]

### WSL Version

2.1.5.0

### Are you using WSL 1 or WSL 2?

- [X] WSL 2
- [ ] WSL 1

### Kernel Version

5.15.146.1-microsoft-standard-WSL2

### Distro Version

Ubuntu 24.04, Ubuntu 22.04, Kali Linux

### Other Software

Cisco AnyConnect Secure Mobility Client 4.10.08029 (this has known issues with WSL2, but I don't seem to be having problems with the networking)

### Repro Steps

Start from a freshly booted computer. 
Launch a WSL Linux distro, or run a WSL.EXE command.
Wait....................
.............................
.............................

### Expected Behavior

I expect it to launch in less than a minute (previously it would launch in under 10 seconds).

### Actual Behavior

I've tried 3 distros (Ubuntu 22.04, Ubuntu 24.04 and Kali Linux) and they all have the same problem. 
Even trying to run wsl.exe commands takes 10+ minutes.

dmesg shows long gaps in time between _hv_balloon_ and _/sbin/ldconfig_, then another gap between some _Ioctl failed_ messages for _dxgk_ and _/sbin/ldconfig.real_

See attached snippit from dmesg output. 

I've fully uninstalled WSL and reinstalled, but the issue persists. 

I've tried re-installing the NVIDIA drivers for Windows because the dxgk errors appear to be related to DirectX.
I tried disabling the GPU (gpuSupport=false) in .wslconfig, and that got rid of the Ioctl failures, but the loading of WSL took even longer.

Is anyone else experiencing this?

```
[    1.731326] EXT4-fs (sdc): mounted filesystem with ordered data mode. Opts: discard,errors=remount-ro,data=ordered. Quota mode: none.
[   49.650182] hv_balloon: Max. dynamic memory size: 16234 MB
[  375.557342] /sbin/ldconfig:
[  375.557345] /usr/lib/wsl/lib/libcuda.so.1 is not a symbolic link

[  375.701335] FS-Cache: Duplicate cookie detected
[  375.701705] FS-Cache: O-cookie c=00000008 [p=00000002 fl=222 nc=0 na=1]
[  375.701948] FS-Cache: O-cookie d=0000000085840207{9P.session} n=0000000035e3d955
[  375.702193] FS-Cache: O-key=[10] '34323934393734383430'
[  375.702372] FS-Cache: N-cookie c=00000009 [p=00000002 fl=2 nc=0 na=1]
[  375.702608] FS-Cache: N-cookie d=0000000085840207{9P.session} n=00000000c5a37aa8
[  375.702859] FS-Cache: N-key=[10] '34323934393734383430'
[  375.875271] misc dxg: dxgk: dxgkio_query_adapter_info: Ioctl failed: -22
[  375.875671] misc dxg: dxgk: dxgkio_query_adapter_info: Ioctl failed: -22
[  375.876001] misc dxg: dxgk: dxgkio_query_adapter_info: Ioctl failed: -22
[  375.876328] misc dxg: dxgk: dxgkio_query_adapter_info: Ioctl failed: -2
[  375.877581] misc dxg: dxgk: dxgkio_query_adapter_info: Ioctl failed: -22
[  375.877915] misc dxg: dxgk: dxgkio_query_adapter_info: Ioctl failed: -22
[  375.878203] misc dxg: dxgk: dxgkio_query_adapter_info: Ioctl failed: -22
[  375.878553] misc dxg: dxgk: dxgkio_query_adapter_info: Ioctl failed: -2
[  739.908389] /sbin/ldconfig.real:
[  739.908392] /usr/lib/wsl/lib/libcuda.so.1 is not a symbolic link
```

**Comment by plasma-vision** [comment]
Well, moments after posting this, I managed to fix it. 
I tried running _wsl.exe --status_, which timed out with a memory error, so I thought to try looking at Windows virtual memory settings (something I haven't thought of for years). 
It was set to "Automatically manage paging file size for all drives". I unchecked that. Chose "no paging file" and clicked set. Then I changed my mind and switched back to "System managed size", and re-enabled "Automatically manage paging file size for all drives" and clicked OK. It asked me to reboot, and now WSL launches quickly every time.
This seems like a totally random fix, if in fact that is what fixed the issue, but if you are having the same problem, give it a try.
I had been trying to fix this for weeks.
